rllib/evaluation/collectors/agent_collector.py

```diff
@@ -265,23 +268,26 @@ def add_action_reward_next_obs(self, input_values: Dict[str, TensorType]) -> Non
 
         for k, v in values.items():
             if k not in self.buffers:
-                if self.training and k.startswith("state_out_"):
+                if self.training and k.startswith("state_out"):
```

Comment:
For now, we need this as an awkward bridge between the old "state_out_..." notation and our new "STATE_OUT" notation. In our future sampling backend, we can switch to "STATE_OUT".